File: cpp/PE006.cpp
#include <iostream>
using namespace std;

long long sumOfSquares(int n)
{
    return n*(n+1)*(2*n+1)/6;
}
long long squareOfSums(int n)
{
    long long a = n*(n+1)/2;
    return a * a;
}
int main()
{
    cout << squareOfSums(100) - sumOfSquares(100) << endl;
    return 0;
}

File: python/PE006.py
#!/usr/bin/env python
def SumOfSquares(n):
    return (n * (n + 1) * (2 * n + 1)) / 6
    
def SquareOfSum(n):
    return (n * n * (n + 1) * (n + 1)) / 4

def main():
    print SquareOfSum(100) - SumOfSquares(100)

if __name__ == '__main__':
    main()
